ts/components/messages/PaymentButton.tsx

```diff
@@ -166,83 +74,125 @@ const getButtonText = (
   });
 };
 
+/**
+ * A component to render the button related to the payment
+ * paired with a message.
+ */
```

Comment:
These are the differences. Should they ave the same look and feel or these changes are being planned in UI review?

| before  | now |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/81377232-5713c980-9105-11ea-9258-d9d8b5b19467.png) | ![now](https://user-images.githubusercontent.com/822471/81377244-5c711400-9105-11ea-8f28-17766f965b49.png)


| before  | now |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/81377338-8f1b0c80-9105-11ea-8530-cb82fc5af6a3.png) | ![now](https://user-images.githubusercontent.com/822471/81377358-99d5a180-9105-11ea-8cbf-2fd3c7013397.png)

Reply:
What concerns the 'paid' button, the differences you highlighted are mostly aimed to have a more coherent design in the message list. Being a button, the 'paid' banner should be equal to the others in terms of dimensions. 

Here I was missing one aspect: the buttons fonts is 16 instead of 14 for all the buttons. I updated the theme so that they have the proper 14 font size.

Here the final result:
<img width="392" alt="image" src="https://user-images.githubusercontent.com/38431762/81415564-e7273280-9148-11ea-8962-0b2f523a2f68.png">